packages/aws-otel-collector/aws-otel-collector.spec

```
@@ -64,11 +66,15 @@ install -d %{buildroot}{%{_cross_bindir},%{_cross_fips_bindir}}
 install -p -m 0755 aws-otel-collector %{buildroot}%{_cross_bindir}
 install -p -m 0755 fips/aws-otel-collector %{buildroot}%{_cross_fips_bindir}
 
+%cross_install_sbom
+
 %files
 %{_cross_attribution_file}
 %{_cross_unitdir}/aws-otel-collector.service
 %{_cross_tmpfilesdir}/aws-otel-collector-tmpfiles.conf
 %{_cross_factorydir}%{_cross_sysconfdir}/aws-otel-collector.yaml
+%{_cross_sbom_package_dir}/%{name}-spdx.json
+%{_cross_sbom_package_dir}/%{name}-cyclonedx.json
```

I would recommend standardizing on just the directory, like we do with `%{_cross_attribution_vendor_dir}`:
```suggestion
%{_cross_sbom_package_dir}
```